kiva/compat.py

```diff
@@ -0,0 +1,22 @@
+"""
+Compatibility module to help support various versions of PIL and
+Pillow.
+
+"""
+from PIL import Image
+
+HAS_FROM_BYTES = hasattr(Image, 'frombytes')
```

Comment:
Since Image is a class from PIL/Pillow I thought that there is no reason to check every time if the frombytes exists, so I thought having a constant with the result was a good idea.